### DISCUSSION TITLE
Will .NET Standard 2.0 Support .NET 8 In the Future

### DISCUSSION THREAD

**Discussion by R-Hurl** [dotnet/runtime#90829] [Q&A] [answered]
Greetings,

Working on a NuGet Package that need to support .NET Framework >= 4.8 and .NET 6 and greater. See on the docs for .NET Standard 2.0 that .NET 7 is supported.

https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0

Will .NET 8 be supported once it is fully released and not in preview?

**Comment by huoyaoyuan** [comment, marked as answer]
There's no action to remove .NET Standard support. All the .NET Standard versions will *be supported by* .NET 8.
